packages/core/components/Modal/CodeSnippet/ConvertToZarr.tsx

```diff
@@ -0,0 +1,307 @@
+import classNames from "classnames";
+import * as React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { TextField, TooltipHost, DirectionalHint } from "@fluentui/react";
+
+import { ModalProps } from "..";
+import BaseModal from "../BaseModal";
+import { SecondaryButton, TertiaryButton } from "../../Buttons";
+import { interaction, selection } from "../../../state";
+import { setConvertFilesSnippet } from "../../../state/interaction/actions";
+import ChoiceGroup from "../../ChoiceGroup";
+
+import styles from "./ConvertToZarr.module.css";
+
+type UIOpts = {
+    destination: string;
+    scenesMode: "all" | "single";
+    sceneIndex: string;
+};
+
+function Snippet({
+    title,
+    text,
+    copyAria,
+}: {
+    title: "Setup" | "Code";
+    text?: string;
+    copyAria: string;
+}) {
+    const [copied, setCopied] = React.useState(false);
+    const value = text ?? "";
+
+    const onCopy = () => {
+        if (!value) return;
+        navigator.clipboard.writeText(value);
+        setCopied(true);
+        window.setTimeout(() => setCopied(false), 900);
+    };
+
+    return (
+        <>
+            <div className={styles.snipHeader}>
+                <div className={styles.snipTitle}>{title}</div>
+                <TooltipHost
```

Comment:
ik this is probably bc CodeSnippet had this TooltipHost as well, but this may not actually show up anymore; `TertiaryButton` now also wraps a `Tooltip` element